projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -1497,4 +1497,5 @@ export interface IClipboardOptions {
 export interface EntityType {
     name: string;
     fields: FieldType[];
+    childEntities?: EntityType[];
```

Comment:
I'd rather keep the EntityType simpler. So I was thinking, can we just have a property like `parent?: string;` & create relations based on this instead of making the entity a heirarchical structure?